src/utils/Utils.ts

```diff
@@ -1341,6 +1341,32 @@ export default class Utils {
     return /^[A-Z0-9- ]*$/.test(plateID);
   }
 
+  public static separateCryptoAlgo(algo: string): CryptoKeyProperties {
```

Comment:
parseCryptoSettings?
parseConfigCryptoSettings?

Reply:
parseConfigCryptoAlgorithm seems more appropiate since I give as input the algorithm from the Crypto Config file and return Crypto Key Properties and not the whole Crypto Setting